specifications/api/swagger.yaml

```diff
@@ -794,11 +786,7 @@ x-responsesPostTransaction: &responsesPostTransaction
 
 x-responsesPostExternalTransaction: &responsesPostExternalTransaction
   <<: *responsesErr400
-  <<: *responsesErr403
   <<: *responsesErr404
-  <<: *responsesErr405
-  <<: *responsesErr406
-  <<: *responsesErr415
```

Comment:
Actually this endpoint can produce codes `405`, `406` and `415`. I will provide tests for that shortly.

Side note: `415` error message is now out-of-date as it reads `I'm really sorry but I only understand 'application/json'...` which is not true for this endpoint which consumes `application/octet-stream`